packages/fuel-indexer-tests/tests/e2e/metrics_postgres.rs

```diff
@@ -1,10 +1,30 @@
+use fuel_indexer_api_server::api::GraphQlApi;
+use fuel_indexer_lib::config::GraphQLConfig;
 use fuel_indexer_postgres as postgres;
-use fuel_indexer_tests::fixtures::{http_client, postgres_connection_pool};
+use fuel_indexer_tests::fixtures::{
+    api_server_app_postgres, http_client, indexer_service_postgres,
+    postgres_connection_pool,
+};
+use tokio::task::spawn;
 
 #[tokio::test]
-#[cfg(all(feature = "e2e", feature = "postgres"))]
+#[cfg(all(feature = "e2e", feature = "postgres", feature = "metrics"))]
```

Comment:
- Is there a reason for this additional `metrics` feature flag? Doesn't seem like something we'd want metrics testing feature flagged.
  - `e2e` is feature flagged because those tests run differently than all other tests
  - `postgres` is feature flagged cause that decides which suite of database tests to run

Reply:
Nah, I was testing a bunch of things and forgot to remove it. 😓